packages/collector/test/tracing/misc/spec_compliance/tracer_compliance_test_cases.json

```diff
@@ -1249,23 +624,114 @@
     },
     "entrySpan.tp": true,
     "entrySpan.lt": "0af7651916cd43dd8448eb211c80319c",
-    "entrySpan.crid": null,
-    "entrySpan.crtp": null,
-    "entrySpan.sy": null,
     "exitSpan.t": "8448eb211c80319c",
     "exitSpan.p": "$new_span_id_1",
     "exitSpan.s": "$new_span_id_2",
-    "exitSpan.ia": null,
-    "exitSpan.tp": null,
-    "exitSpan.lt": null,
-    "exitSpan.crid": null,
-    "exitSpan.crtp": null,
-    "exitSpan.sy": null,
     "X-INSTANA-T out": "8448eb211c80319c",
     "X-INSTANA-S out": "$new_span_id_2",
     "X-INSTANA-L out": "1",
     "traceparent out": "00-0af7651916cd43dd8448eb211c80319c-$new_span_id_2-01",
     "tracestate out": "in=8448eb211c80319c;$new_span_id_2,member01=value01,member02=value02,member03=value03,member04=value04,member05=value05,member06=value06,member07=value07,member08=value08,member09=value09,member10=value10,member11=value11,member12=value12,member13=value13,member14=value14,member15=value15,member16=value16,member17=value17,member18=value18,member19=value19,member20=value20,member21=value21,member22=value22,member23=value23,member24=value24,member25=value25,member26=value26,member27=value27,member28=value28,member29=value29,member30=value30,member31=value31",
     "index": 35
+  },
+  {
```

Comment:
The main motivation for this PR are the 5 new test cases that follow here (test cases 36 - 40). This JSON file is a direct copy of the same JSON file here: https://github.ibm.com/instana/tracer-test-suite/tree/main/test-cases